recipe/meta.yaml

```diff
@@ -7,7 +7,7 @@ package:
   version: "{{ version }}"
 
 source:
-  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name|capitalize }}-{{ version }}.tar.gz"
```

Comment:
I'd better recommend to change the `{% set name = "yapsy" %}` on line 1 to have `Yapsy` there. The `name` var is already lowecased.

Reply:
The directories {{ name[0] }} and {{ name }} are not capitalized so those would have to become {{ name[0]|lower }} and {{ name|lower }}.

Reply:
Interesting. It seems to work for `ModestImage`: https://github.com/nsls-ii-forge/modestimage-feedstock/blob/4da15e709d1be189ddfa1e060e5379bbc2c84827/recipe/meta.yaml#L10.

Reply:
I was confused by the duplicated "name" variables. The duplication of variables by the {% set... %} lines and the yaml seems unnecessary. What is the advantage?

Reply:
The `{% set name = ... %}` variable is used by jinja to expand the names in the template.
The `package: --> name: ...` is used by conda to name the resulted package. And in this case it's also templated.

In other words, the `{% set ... %}` things do not appear in the final recipe after its rendering by jinja. Here is an example of the modestimage package's contents:
![image](https://user-images.githubusercontent.com/13209176/67953516-a8c32080-fbc5-11e9-90d3-5d862f4bf3cd.png)